test/unit/BPool/BPool_Unbind.tree

```diff
@@ -0,0 +1,23 @@
+BPool::Unbind
+├── when reentrancy lock is set
+│   └── it should revert
+├── when caller is NOT controller
+│   └── it should revert
+└── when caller is controller
+    ├── when token is not bound
+    │   └── it should revert
+    └── when token is bound
```

Comment:
inconsistent branching with Bind, there you have `when token is bound`, and right below `when pool is finalized`, in here you're branching into `when token is bound`, i'd avoid it

Reply:
I don't think it's avoidable while still using the modifiers as `bulloak` intends

in `unbind`, `when token is bound` is a *precondition* for all of the subtree under it, and we actually take advantage of it by reusing the modifier

in `bind`, `when token is bound` only has one leaf under it, and is not a required setup for anything else, so it doesn't get a modifier as there's nothing to reuse.

I could decouple setup/preconditions from modifiers, but I think it'd have greater cognitive overhead, as you'd have to figure out how tests are set up from the code, without guidance from the `.tree`